views/js/pciCreator/dev/audioRecordingInteraction/runtime/audioRecordingInteraction.js

```diff
@@ -719,6 +778,10 @@ define([
                 promises.push(self.recorder.destroy());
             }
 
+            if (self._delayCallback) {
+                promises.push(self._cleanDelayCallback);
```

Comment:
What is the purpose of this?
If it is to give the refence of the function as a result of the promise... it can't work, the method needs a valid lexical scope to work properly.
Otherwise, it lacks the parenthesis to be properly called. If so, why is it added as a promise? the method is synchronous and doesn't return anything...

Reply:
several errors in just one line! 🤦‍♂️
No need to add as a promise. I will call the function directly.